components/user_list_row/index.ts

```diff
@@ -5,9 +5,17 @@ import {connect} from 'react-redux';
 
 import {getStatusForUserId} from 'mattermost-redux/selectors/entities/users';
 
+import {GlobalState} from 'types/store';
+
+import {UserProfile} from '@mattermost/types/users';
+
 import UserListRow from './user_list_row';
 
-function mapStateToProps(state, ownProps) {
+type OwnProps = {
+    user: UserProfile;
```

Comment:
Since making this required isn't causing any type errors, would you mind removing the `|| {}` where this is used on line 19? I double checked as well to make sure there's no plain JS that might be passing in null by mistake

Reply:
Good point, thanks